### PR TITLE
Add support for waist circumference

### PR DESCRIPTION
**Pull request by kabaros**
## Description

This PR adds support for [waist circumference](https://developer.apple.com/documentation/healthkit/hkquantitytypeidentifierwaistcircumference/) to `react-native-health` library.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have checked my code and corrected any misspellings

## Screenshot
image

### CODE REVIEW

**Comment by AngelinCalu** [comment]
Any ETA about when this is going to be reviewed/merged?

**Comment by Macelai** [comment]
> Any ETA about when this is going to be reviewed/merged?

Sorry for the delay, I am having some problems about some packages used by the project. I'll give another try over the weekend

**Review by Macelai — APPROVED** [review]
Great job! Thanks for your time!

File: RCTAppleHealthKit/RCTAppleHealthKit+TypesAndPermissions.m [hunk 1]
```diff
@@ -27,6 +27,8 @@ - (nullable HKObjectType *)getReadPermFromText:(nonnull NSString*)key {
         return [HKObjectType characteristicTypeForIdentifier:HKCharacteristicTypeIdentifierBiologicalSex];
     } else if ([@"BloodType" isEqualToString: key]) {
         return [HKObjectType characteristicTypeForIdentifier:HKCharacteristicTypeIdentifierBloodType];
+    } else if ([@"WaistCircumference" isEqualToString: key] && systemVersion >= 11.0) {
```
Comment (lucaspbordignon) [inline RCTAppleHealthKit/RCTAppleHealthKit+TypesAndPermissions.m:30]: Great catch on iOS 11 check! 🚀

**Review by lucaspbordignon — APPROVED** [review]
Thanks for the contribution @kabaros!

Merging now and will be included in the next release 🚀

**Pull request merged**